packages/core/chrome/core-chrome-browser-internal/src/ui/header/header_logo.tsx

```diff
@@ -102,11 +119,12 @@ export function HeaderLogo({ href, navigateToApp, loadingCount$, ...observables
         <img
           src={customizedLogo}
           className="chrHeaderLogo__mark"
```

Comment:
@eokoneyo thanks for taking a look at this. I would prefer to leave this as it is. It will help this PR, and others like it, to avoid ballooning the refactoring into something larger than it needs to be. Changing this code could impact other code owners as well, and then add the need for more teams to review.

Reply:
bb458511019 replaced a className with a testId, since it was only needed for a test

Reply:
@logeekal The downside of using classNames for non-style purposes is that _classNames are not a public contract_, so they shouldn't be used for other things.

As we're replacing CSS/SASS with Emotion, for thoroughness, the typical followup will be to remove the className from the DOM, for thoroughness. We could break integrations if we fail to realize that some place in the repo is using that className as a selector in production code. 

Relevant: https://github.com/elastic/kibana/pull/199885/commits/24f6977030a96b96a9e59072bd991ba33127e396